tests/BiosTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+use Laracasts\TestDummy\Factory;
+
+class BiosTest extends IntegrationTestCase
+{
+    /** @test */
+    function user_can_create_a_private_bio()
+    {
+        $user = Factory::create('user');
+
+        $this->actingAs($user)
+            ->visit('/bios/create')
+            ->type('Some Nickname', '#nickname')
+            ->type('A big chunk of bio-friendly text', '#body')
+            ->select('no', '#public')
+            ->press('Create');
+            //->seePageIs('bios'); //not sure how to test the string /bios/x
+
+        $this->seeInDatabase('bios', [
+            'nickname' => 'Some Nickname',
+            'body' => 'A big chunk of bio-friendly text',
+            'public' => 'no'
```

Comment:
This passes for me as 'no' but fails as 0, '0', false, 'false', 1, 'yes', 'random'.
Not sure where that may be originating from.

I can assert manually that creating that results in a 0 in the database, but for some reason only 'no' will pass.